### PR TITLE
[MIG] stock_request_template: Migration to 13.0

### PR DESCRIPTION
**Pull request by albariera**
Test added

### CODE REVIEW

File: stock_request_template/models/stock_request_template.py [hunk 1]
```diff
@@ -78,41 +74,32 @@ class StockRequestTemplateLine(models.Model):
...
-    product_id = fields.Many2one(
-        "product.product",
-        "Product",
-        domain=[("type", "in", ["product", "consu"])],
-        ondelete="restrict",
-        required=True,
-    )
 
-    @api.constrains("product_id")
-    def _check_product_uom(self):
```
Comment (etobella) [inline stock_request_template/models/stock_request_template.py]: why did you remove this?

Comment (albariera) [reply stock_request_template/models/stock_request_template.py]: Because I thought it was not necessary. The uom field has already a domain that only lets the user choose between the uoms of the product uom category, so the user can not put it incorrectly. Please correct me If I am wrong!
